lib/svg.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const svgoLoader = {
+    loader: 'svgo-loader',
```

Comment:
Something to discuss is if we use `require.resolve` when using a loader or not. This is something that should be done for all plugins and not just this one.

The difference is that if we use a `string`, webpack will load the loader from the root `node_modules` which might not be this done exactly, depending on the dedup.